src/main/java/org/broadinstitute/consent/http/models/FileStorageObject.java

```diff
@@ -5,10 +5,26 @@
 
 import java.io.InputStream;
 import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Objects;
 
 public class FileStorageObject {
 
+    public static final String QUERY_FIELDS_WITH_FSO_PREFIX =
```

Comment:
Would this be better suited in the DatasetDAO?  It looks like that's the only place this is used.
Consider @rushtong 's comment about using text blocks.  He left that one on a PR of mine too and I think it really does help with readability.

Reply:
Agreed on the multi-line string: seems to be a new pattern in Consent which I like. As far as keeping this here or in the dataset dao, I think it should be kept here, because it's intended use is across Consent. If we added FSO objects to any other objects, they would need this string. Plus, there are other examples of other models with a similar static constant on them.